tasks/extract-binaries-from-image/README.md

```diff
@@ -0,0 +1,14 @@
+# extract-binaries-from-image
+
+Tekton task that extracts binaries to be released on github.com from an image.
+
+The path to the directory inside the provided workspace where the binaries were
```

Comment:
The result was removed so this should be removed

Reply:
Removed!